bundler/lib/dependabot/bundler/update_checker.rb

```diff
@@ -116,8 +118,19 @@ def conflicting_dependencies
         )
       end
 
+      sig { returns(PackageManagerBase) }
+      def package_manager
```

Comment:
We already have a `ecosystem_versions` on the FileFetcher that we use to [POST the ecosystem versions](https://github.com/dependabot/dependabot-core/blob/a33e69ce959a90fdaf7a46c20907431ba5a50c15/updater/lib/dependabot/api_client.rb#L197-L213) to the API to gather usage statistics.

https://github.com/dependabot/dependabot-core/blob/30ffbccf4ba0bbf0d22a6f65f162a18a2d77d339/common/lib/dependabot/file_fetchers/base.rb#L180

We should either reuse that functionality or move it into the FileParser which might be the best place for it, and perhaps reconcile the differences between what they all return since they're all doing the same thing.